src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -0,0 +1,42 @@
+"""
+Management utility to create superusers.
+
+Inspired by django.contrib.auth.management.commands.createsuperuser.
+(http://bit.ly/2cTgsNV)
+"""
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.core import exceptions
+from django.core.management.base import BaseCommand
+from django.utils.six.moves import input
+from django.utils.text import capfirst
+
+
+class Command(BaseCommand):
+    help = 'Used to create a superuser.'
+    required_fields = ['username', 'email']
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.UserModel = get_user_model()
```

Comment:
I'd rather see the `get_user_model` call twice than saving this on the command instance. Mostly style reason but it also creates unneccessary (possible) side-effects e.g in case of mocking.